### DISCUSSION TITLE
"Is in" condition against null works incorrectly

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#14604] [Help] [answered]
While testing my app on the cloud (v2.32.5), I've just found:

1. "1" is in "1,2,3" evaluates to true
2. "1" is in "2,3" evaluates to false
3. "1" is in **null**, **[]**, or **""** evaluates to true (!)

I think this is clearly a bug, am I right :)

**Comment by mikesealey** [comment, marked as answer]
Hey @dannyshin76 I've had a closer look and I think you're right. I've submitted an issue here: https://github.com/Budibase/budibase/issues/14605

Please feel free to add any addition information you think might be relevant.

**Reply by dannyshin76** [reply]
I see this bug is gone on v2.32.7. Thank you!
